test/token/ERC721/ERC721.behavior.js

```diff
@@ -30,44 +23,30 @@ function shouldBehaveLikeERC721() {
     });
 
     describe('balanceOf', function () {
-      describe('when the given address owns some tokens', function () {
-        it('returns the amount of tokens owned by the given address', async function () {
-          expect(await this.token.balanceOf(this.owner)).to.equal(2n);
-        });
+      it('when the given address owns some tokens', async function () {
```

Comment:
I'd combine both the `describe` text and the `it` text so this is clear on its intention. Otherwise, we will only know the situation the test is describing but we won't know the intended result. Although the test is pretty straight forward (`expect(...).to.equal(...)`) I think we prefer to be explicit.

```suggestion
      it('returns the amount of tokens owned by the given address when the given address owns some tokens', async function () {
```

Same for all others